web-common/src/features/entity-management/file-selectors.ts

```diff
@@ -7,6 +7,7 @@ import { createRuntimeServiceListFiles } from "@rilldata/web-common/runtime-clie
 export function useMainEntityFiles(
   instanceId: string,
   prefix: "sources" | "models" | "dashboards" | "charts" | "custom-dashboards",
+  transform = (name: string) => name,
```

Comment:
Rather than this transform parameter, can we instead just edit the `createRuntimeServiceListFiles` selector function to return file paths not file names?